streaming/base/storage/download.py

```diff
@@ -272,6 +273,29 @@ def download_from_oci(remote: str, local: str) -> None:
     os.rename(local_tmp, local)
 
 
+def download_from_hf(remote: str, local: str) -> None:
+    """Download a file from remote Hugging Face to local.
+
+    Args:
+        remote (str): Remote path (Hugging Face).
+        local (str): Local path (local filesystem).
+    """
+    from huggingface_hub import hf_hub_download
+
+    if 'hf://' not in remote:
+        raise ValueError(f'Expected remote path to start with `hf://`, got {remote}.')
+
+    _, _, _, repo_org, repo_name, path = remote.split('/', 5)
```

Comment:
Do [huggingface_hub](https://github.com/huggingface/huggingface_hub) have an API to fetch `repo_org`, `repo_name`, `path`?

Reply:
Do you mean to auto-parse this? Not that I'm aware of.  TBH, I wasn't aware that they had the fspec setup until I read the issue. This is the only documentation I'm aware of: https://huggingface.co/docs/huggingface_hub/guides/hf_file_system